LoopTestingKit/TestingScenario.swift

```diff
@@ -18,7 +18,8 @@ public struct TestingScenario {
     public func instantiate(relativeTo referenceDate: Date = Date()) -> TestingScenarioInstance {
         let glucoseSamples = dateRelativeGlucoseSamples
             .map { $0.newGlucoseSample(relativeTo: referenceDate) }
-            .filter { $0.date <= referenceDate }
+        let pastGlucoseSamples = glucoseSamples.filter { $0.date <= referenceDate }
+        let futureGlucoseSamples = glucoseSamples.filter { $0.date > referenceDate }
```

Comment:
It looks like the glucoseSamples already filter based on the referenceDate? Should `dateRelativeGlucoseSamples` be used?

Reply:
The `DateRelativeGlucoseSample` is just a structured type used in the scenario file and needs to be converted to a `NewGlucoseSample` with the dateOffset converted to a relative date which is what the line above this does.

Reply:
Oh... just realized I mis-read. all good.